storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -915,9 +917,9 @@ static dberr_t open_or_create_log_file(bool *log_file_created, ulint i,
     /* Create in memory the file space object
     which is for this log group */
 
-    *log_space = fil_space_create(name, dict_sys_t::s_log_space_first_id,
+    *log_space = fil_space_create(name, dict_sys_t::s_log_space_id,
```

Comment:
Hmm, looks like you answered my question above, so we will create fil_sapce_t object for redo log files.
You shouldn't do this because now redo file are multiple files and each one of them is like a different tablespace.

Reply:
not present in latest commit.

Reply:
Looks l like rewrote this part